opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -180,7 +180,10 @@ namespace Opm
 
             // write the inital state at the report stage
             if (timer.initialStep()) {
-                output_writer_.writeTimeStep( timer, state, well_state );
+                // No per cell data is written for initial step, but will be
```

Comment:
This comment is good, and should be duplicated in the other places this code is duplicated. That also makes it easier to compare/refactor/un-duplicate it later.